### ISSUE TITLE
Workflows dont show job templates using vaults as their credential

### ISSUE THREAD

**Issue by jmighion** [ansible/awx#1905]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY

When choosing the template for a workflow to run, the list does not populate if the template's credential is set to vault. The current documentation states 
```
You will not be able to select job templates that don't have a default inventory or credential when populating a workflow graph.
```

##### ENVIRONMENT
* AWX version: 3.2.2
* AWX install method: tower bundled installer
* Ansible version: 2.5.2
* Operating System: RHEL
* Web Browser: Chrome

##### STEPS TO REPRODUCE

Create a standard job template but for the credential use a vault credential.

##### EXPECTED RESULTS

Be able to select template from workflow

##### ACTUAL RESULTS

Unable to select template from workflow

**Comment by AlanCoding** [comment]
I cannot reproduce this. Ever since the multi-credential changes landed, a machine credential has not been required. I can add JTs to workflows with no credential, or with a vault credential and nothing else. You cannot add JTs that use credentials which need passwords provided.

Ansible Tower documentation does not necessarily apply to AWX use, and this is an example of that. AWX development has changed how this works, and the Ansible Tower function and documentation will eventually change to reflect this as well.

**Comment by wenottingham** [comment]
Closing as fixed in AWX, will be in 3.3
